src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/PackageDeclarationCheckTest.java

```diff
@@ -56,17 +56,6 @@ public void testDefaultWithPackage() throws Exception {
         verify(checkConfig, getPath("InputPackageDeclarationPlain.java"), expected);
     }
 
-    @Test
-    public void testOnFileWithCommentOnly() throws Exception {
-        final DefaultConfiguration checkConfig = createModuleConfig(PackageDeclarationCheck.class);
-
-        final String[] expected = {
-            "1: " + getCheckMessage(MSG_KEY_MISSING),
-        };
-
-        verify(checkConfig, getPath("InputPackageDeclarationWithCommentOnly.java"), expected);
```

Comment:
please keep it. On such files there should no violations too.

Reply:
But this test is failing due to this change, a file with comment does not generate a violation. Are you suggesting that I should change the `expected` violations to no violations?

Reply:
it should not generate violation - it is by design.
Tests is good to show that.

Reply:
Done

Reply:
please restore test method in original place, not need to remove it and add it at other place.
We need to minimize diff.

Reply:
Done